source/ut_utils.pks

```diff
@@ -11,10 +11,10 @@ create or replace package ut_utils is
     tr_failure - one or more asserts failed
     tr_error   - exception was raised
   */
-  tr_success     constant number(1) := 1; -- test passed
-  tr_failure     constant number(1) := 2; -- one or more asserts failed
-  tr_error       constant number(1) := 3; -- exception was raised
-	
+  tr_success          constant binary_integer := 1; -- test passed
+  tr_failure          constant binary_integer := 2; -- one or more asserts failed
+  tr_error            constant binary_integer := 3; -- exception was raised
```

Comment:
yes.. these constants shoudl be of teh same type as ut_assert_result.result  (or vise versa)

and no.. `tr_success     constant ut_assert_result.result%TYPE := 1;` doesnt work  :(